.github/workflows/pull_requests.yml

```diff
@@ -279,7 +279,7 @@ jobs:
             ${{ runner.os }}-go-integration-
       - name: Run integration test - ${{ matrix.test_name }}
         working-directory: ./tests/integration/
-        run: make test TEST_NAME=${{matrix.test_name}} KIND_NODE_IMAGE=${{matrix.kind_image}}
+        run: make test TEST_NAME=^${{matrix.test_name}}$ KIND_NODE_IMAGE=${{matrix.kind_image}}
```

Comment:
I think we're saying here to match for that test name from the beginning to the end of the string? Why couldn't we just specify the test name instead?

Reply:
nvm I got it
```if we have a test whose name is a prefix of another test, both will be run```

Reply:
@swiatekm-sumo How do you explain this?

```
➜  helm git:(drosiek-fix-non-lower-case-fields) REGENERATE_GOLDENFILES=0 go test -run="^TestGoldenFiles/testdata/goldenfile/terraform/custom.input.yaml$" -v              
=== RUN   TestGoldenFiles
=== RUN   TestGoldenFiles/testdata/goldenfile/terraform/custom.input.yaml
=== PAUSE TestGoldenFiles/testdata/goldenfile/terraform/custom.input.yaml
=== RUN   TestGoldenFiles/testdata/goldenfile/terraform_custom/custom.input.yaml
=== PAUSE TestGoldenFiles/testdata/goldenfile/terraform_custom/custom.input.yaml
=== CONT  TestGoldenFiles/testdata/goldenfile/terraform/custom.input.yaml
=== CONT  TestGoldenFiles/testdata/goldenfile/terraform_custom/custom.input.yaml
--- PASS: TestGoldenFiles (0.00s)
    --- PASS: TestGoldenFiles/testdata/goldenfile/terraform_custom/custom.input.yaml (0.28s)
    --- PASS: TestGoldenFiles/testdata/goldenfile/terraform/custom.input.yaml (0.28s)
PASS
ok      github.com/SumoLogic/sumologic-kubernetes-collection/tests/helm 0.300
```

why `TestGoldenFiles/testdata/goldenfile/terraform_custom/custom.input.yaml` matches `"^TestGoldenFiles/testdata/goldenfile/terraform/custom.input.yaml$"`? :confused: